sql/conseil.sql

```diff
@@ -555,13 +552,14 @@ ALTER TABLE ONLY tezos.operations
 --
 
 CREATE INDEX baking_rights_level_idx ON tezos.baking_rights USING btree (level);
-
+CREATE INDEX ix_delegate_priority ON tezos.baking_rights USING btree (delegate, priority);
 
 --
 -- Name: endorsing_rights_level_idx; Type: INDEX; Schema: tezos; Owner: -
 --
 
 CREATE INDEX endorsing_rights_level_idx ON tezos.endorsing_rights USING btree (level);
+CREATE INDEX ix_delegate_slot ON tezos.endorsing_rights USING btree (delegate);
```

Comment:
Is this actually correct? The name of the index contains `slot` but there is no slot in the actual definition.